### DISCUSSION TITLE
Using same keys for different accounts (problem)

### DISCUSSION THREAD

**Discussion by Sinkler521** [clerk/javascript#3986] [Support request - DO NOT OPEN HERE]
### We use Clerk for authentication in the project (**a few people working with one account**). We got such problem trying to log in:

console: 
_🔒 Clerk: The SignIn or SignUp modals do not render when a user is already signed in, unless the application allows multiple sessions. Since a user is signed in and this application only allows a single session, this is no-op. (This notice only appears in development)_

Even the auth window doesn't render so in a webpage nothing happens.

Apparently, it seems impossible to use one keys pair (public + private) and keep the project turned on for more than one person. I seek a solution now, will it help if create one keys pair (I'm not sure we can do this, except private key because in Clerk private one can be created). Or, please, if you faced such problem somewhen, how did you solve?

I've tried to create and use different private key (unlike public key, a private one can be created). That doesn't help

**Comment by jescalan** [comment]
https://github.com/orgs/clerk/discussions/3829
